_pytest/terminal.py

```diff
@@ -20,10 +20,15 @@ def pytest_addoption(parser):
     group._addoption('-q', '--quiet', action="count",
                dest="quiet", default=0, help="decrease verbosity."),
     group._addoption('-r',
-         action="store", dest="reportchars", default=None, metavar="chars",
+         action="store", dest="reportchars", default='', metavar="chars",
          help="show extra test summary info as specified by chars (f)ailed, "
               "(E)error, (s)skipped, (x)failed, (X)passed (w)pytest-warnings "
```

Comment:
Nice, but also I think we should remove `(w)pytest-warnings` from the list as it no longer has an effect.